README.md

```diff
@@ -80,6 +80,12 @@ Example usage:
 
 # print list of formated rules for use with `pending` in config file
 ./node_modules/.bin/ember-template-lint app/templates/application.hbs --print-pending
+
+# specify custom ignore pattern `['**/dist/**', '**/tmp/**', '**/node_modules/**']` by default
+./node_modules/.bin/ember-template-lint /tmp/template.hbs --ignore-pattern "**/foo/**" "**/bar/**"
+
+# disable ignore pattern entirely
+./node_modules/.bin/ember-template-lint /tmp/template.hbs --no-ignore-pattern
```

Comment:
I know either way works, but mind putting the options before the files to lint?

```suggestion
./node_modules/.bin/ember-template-lint --no-ignore-pattern /tmp/template.hbs 
```

Reply:
hm, wouldn't it be inconsistent with all the other examples?

Reply:
Yikes! You are right, lets fix this in a separate PR...